source/settings/command-line-options.txt

```diff
@@ -116,6 +116,18 @@ value in the :guilabel:`Settings` panel.
        
        To learn more, see :ref:`compass-enable-dev-tools`.
 
+   * - .. option:: --enableGenAIFeatures
+     - Allow |compass-short| to use generative AI for natural language querying.
+       This feature requires an Atlas login and a deployed Atlas cluster.
+
+   * - .. option:: --enableGenAISampleDocumentPassing
+     - Allow |compass| to send sample field values with query and
+       aggregation generation requests. These values can improve
+       the results from the AI.
+
+       If set, this option implicitly sets the
+       :option:`--enableGenAIFeatures` option.
```

Comment:
This should mirror the text used below. It only sets the other option if the value is true.

Reply:
Fixed